styles.css

```diff
@@ -1 +1,63 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+.grid-container {
+    background-color: hsl(25, 18%, 44%);
+    display: grid;
+    grid-gap: 16px;
+    grid-template-columns: 1fr;
+    grid-template-rows: 50px auto 50px 50px 50px 50px 50px;
+    grid-template-areas:
+            "header"
+            "content-b"
+            "content-c"
+            "aside-d"
+            "aside-e"
+            "."
+            "footer";
+}
+
+.grid-item {
```

Comment:
Je bent de padding vergeten in de container en je hebt grid-gap gebruikt i.p.v. column en row. Voor het uiteindelijke resultaat maakt het niets uit maar ik weet niet hoe het uitpakt als het ingewikkelder wordt.